lib/classifier-reborn/extensions/hasher.rb

```diff
@@ -27,7 +27,7 @@ def clean_word_hash(str, language = 'en', enable_stemmer = true)
     def word_hash_for_words(words, language = 'en', enable_stemmer = true)
       d = Hash.new(0)
       words.each do |word|
-        next unless word.length > 2 && !STOPWORDS[language].include?(word)
+        next unless word.length > 0 && !STOPWORDS[language].include?(word)
```

Comment:
I am afraid that this will start white-listing tokens of one and two characters which is not the current behavior. This can be better dealt with the #131 proposal.